content/_includes/server-config.md

```diff
@@ -25,6 +25,11 @@ tls:
   cert:     ./server.cert  # Path to the TLS certificate
   password: ""             # An optional password to decrypt the TLS private key
   
+  # Specify how/whether the KES server verifies certificates presented
+  # by clients. Valid values are "on" and "off". Defaults to off, which
+  # is recommended for most use cases.
+  auth:     ""
```

Comment:
We're saying valid values are `on` or `off`, but then the provided value is `""`. Does that imply the default?

Reply:
From https://github.com/minio/kes/blob/53b74e38697bc68fd88dff7a3cf431db692db9ef/kesconf/config.go#L257C87-L257C87, empty and `off` have the same effect.